integration/traceloop_test.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"os"
+	"os/exec"
+	"regexp"
+	"testing"
+	"text/template"
+
+	"github.com/kr/pretty"
+)
+
+var integration = flag.Bool("integration", false, "run integration tests")
+
+// image such as docker.io/kinvolk/gadget:latest
+var image = flag.String("image", "", "gadget container image")
+
+func TestTraceloop(t *testing.T) {
+	if !*integration {
+		t.Skip("skipping integration test.")
+	}
+
+	commands := []struct {
+		name           string
+		cmd            string
+		outputName     string
+		expected       string
+		expectedRegexp string
+		ignoreOutput   bool
+		debugFailed    bool
+	}{
+		{
+			name:         "Cleanup test namespace from previous tests",
+			cmd:          "kubectl delete ns test-ig --force --grace-period=0 || true",
+			ignoreOutput: true,
+		},
+		{
+			name:         "Cleanup gadget deployment from previous tests",
+			cmd:          "$KUBECTL_GADGET deploy $GADGET_IMAGE_FLAG | kubectl delete --force --grace-period=0 -f - || true",
+			ignoreOutput: true,
+		},
+		{
+			name:     "Deploy Inspektor Gadget",
+			cmd:      "$KUBECTL_GADGET deploy $GADGET_IMAGE_FLAG | kubectl apply -f -",
+			expected: "serviceaccount/gadget created\nclusterrolebinding.rbac.authorization.k8s.io/gadget created\ndaemonset.apps/gadget created\n",
```

Comment:
I am not sure if we rely on that output. Couldn't we just check the return value, ignore the output and let the next test take care?

Reply:
Ok. I could also write a regexp with just "gadget created".